sql/pre_install/tables.sql

```diff
@@ -191,6 +191,8 @@ CREATE TABLE IF NOT EXISTS _timescaledb_catalog.installation_metadata (
 SELECT pg_catalog.pg_extension_config_dump('_timescaledb_catalog.installation_metadata', $$WHERE key='exported_uuid'$$);
 
 -- Set table permissions
+GRANT SELECT ON ALL TABLES IN SCHEMA _timescaledb_cache TO PUBLIC;
 GRANT SELECT ON ALL TABLES IN SCHEMA _timescaledb_catalog TO PUBLIC;
 GRANT SELECT ON ALL TABLES IN SCHEMA _timescaledb_config TO PUBLIC;
-GRANT SELECT ON _timescaledb_internal.bgw_job_stat TO PUBLIC;
+GRANT SELECT ON ALL TABLES IN SCHEMA _timescaledb_internal TO PUBLIC;
```

Comment:
My understanding is that the original reason these tables were not given public permissions is because they are considered "internal" schema, and it's not necessary for users to be able to read them. Is there a way to only grant these privileges right before a `pg_dump`?

Reply:
If they are supposed to be truly internal then they shouldnt be marked as dumpable.

Reply:
And if they are not marked as dumpable then we need to be able to recreate that state from other information.